lib/manager/common.ts

```diff
@@ -47,7 +47,8 @@ export interface UpdateArtifactsConfig extends ManagerConfig {
   postUpdateOptions?: string[];
   ignoreScripts?: boolean;
   updateType?: UpdateType;
-  toVersion?: string;
+  newValue?: string;
```

Comment:
Coincedence?

Reply:
Yes, I refactored the gradle-wrapper artifacts to use newValue instead of toVersion